### DISCUSSION TITLE
Plans to add 'spherical_length' in v2.0?

### DISCUSSION THREAD

**Discussion by mboeringa** [osm2pgsql-dev/osm2pgsql#2177] [General]

**Comment by joto** [comment]
I am not aware of any such plans. Usually we include functionality if and when somebody needs it, not before that.

**Comment by mboeringa** [comment]
> Usually we include functionality if and when somebody needs it, not before that.

Well, since the pull request that added 'spherical_area' (https://github.com/osm2pgsql-dev/osm2pgsql/pull/2019) did not reference an existing open issue or discussion / question, I guess you added it based on a gut feeling that it might be useful to some people (which I agree it is).

My similar gut feeling says this one is likely of equal value ;-)

**Reply by joto** [reply]
Plenty of things never make it into issues. This is a silly discussion. Lets stop that.
